server/server.go

```diff
@@ -301,6 +316,10 @@ func copyHeader(srcHeaders http.Header, dstHeaders http.Header) {
 // readClusterIDsForOrgID reads the list of clusters for a given
 // organization from aggregator
 func (server HTTPServer) readClusterIDsForOrgID(orgID types.OrgID) ([]types.ClusterName, error) {
+	if server.amsClient != nil {
+		return server.amsClient.GetClustersForOrganization(orgID, nil, []string{"Deprovisioned", "Archived"}), nil
```

Comment:
we are gonna to use that code multiple times, so what about making those two strings constants?

Reply:
Agree